demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -156,10 +157,18 @@ def pack_id(self) -> str:
 
     @property
     def ignored_errors(self) -> List[str]:
-        return self.get_ignored_errors(PACK_METADATA_FILENAME)
-
-    def ignored_errors_related_files(self, file_path: Union[str, Path]) -> List[str]:
-        return self.get_ignored_errors((Path(file_path)).name)
+        ignored_errors = self.get_ignored_errors(PACK_METADATA_FILENAME)
+        if not ignored_errors:
+            file_path = safe_get_relative_path(self.path, CONTENT_PATH)
+            ignored_errors = self.get_ignored_errors(file_path / PACK_METADATA_FILENAME)
+        return ignored_errors
+
+    def ignored_errors_related_files(self, file_path: Path) -> List[str]:
+        ignored_errors = self.get_ignored_errors((Path(file_path)).name)
+        if not ignored_errors:
+            file_path = safe_get_relative_path(file_path, CONTENT_PATH)
+            ignored_errors = self.get_ignored_errors(file_path)
+        return ignored_errors
```

Comment:
```suggestion
        if ignored_errors := self.get_ignored_errors((Path(file_path)).name):
            return ignored_errors
         file_path = safe_get_relative_path(file_path, CONTENT_PATH)
         return self.get_ignored_errors(file_path)
```

Looks like you have 4 functions that do very similar logic, maybe use the same logic in a func and call it from each method?